CMakeLists.txt

```diff
@@ -540,7 +540,6 @@ if(BUILD_TESTS)
     NAME late_joiners
```

Comment:
Could we rename this test to `late_joiners_pbft` as it is the only e2e test that is PBFT specific?

Reply:
From my understanding this should not be PBFT specific but this catches the known RAFT bug?

Reply:
yes exactly

Reply:
I'll change it to `late_joiners_pbft` for now and we can add one for raft later